afsbirez/settings/dev.py

```diff
@@ -21,4 +22,9 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-INSTALLED_APPS.append('django_extensions')
\ No newline at end of file
+INSTALLED_APPS.append('django_extensions')
+
+REST_PROXY = {
```

Comment:
That's actually a stopgap until I get a mock version of the SAM API set up.  Once I do, `dev` can lose the credentials entirely and just work against a mock.